tasks/install.yml

```diff
@@ -1,9 +1,57 @@
 ---
 # ©Copyright 2015 Hewlett-Packard Development Company, L.P.
 
+- name: Include system dependencies
+  include_vars: "vars{{ ansible_os_family }}.yml"
+
+- name: Include DB dependencies
+  include_vars: "{{ ansible_os_family|lower }}_{{ database_backend }}.yml"
+  when: database_support
+
+- name: Fetch pip getter
+  get_url: url="{{ get_pip_download_url }}/get-pip.py" dest={{ download_tmp_dir }}/ timeout=60
+
+- name: Install pip
+  command: "python {{ download_tmp_dir }}/get-pip.py"
+
+- name: Install deps to avoid pip doing compilation
+  yum: name={{item}} state=present
+  with_items: dependencies
+  when: not skip_install and ansible_os_family == "RedHat"
+
 - name: Install deps to avoid pip doing compilation
   apt: name={{item}} state=present
   with_items: dependencies
+  when: not skip_install and ansible_os_family == "Debian"
+
+- name: Install DB dependencies (RedHat)
+  yum: name={{item}} state=present
+  with_items: database_dependencies
+  when: not skip_install and ansible_os_family == "RedHat" and (install_yum_postgres_lib is defined and install_yum_postgres_lib)
+
+- name: Install DB dependencies (Debian)
+  apt: name={{item}} state=present update_cache=yes
+  with_items: database_dependencies
+  when: not skip_install and ansible_os_family == "Debian"
+
+- include: pip_index.yml
+  when: not skip_install
+
```

Comment:
Same here this pip/virtualenv setup should be done in another role or the playbook directly.